### DISCUSSION TITLE
Error

### DISCUSSION THREAD

**Discussion by sophiastr** [ho-dev/HattrickOrganizer#1488] [General]
![image](https://user-images.githubusercontent.com/72817947/166006500-3ad0f0ba-ae9c-46b6-8bd1-268901e62e6c.png)
I was working normally before I pull, but as soon as I got the new changes I got this error. How can I fix it?
I need to work on the project.

**Comment by tychobrailleur** [comment]
Is it possible you have merge issues in your local repository from your recent git pull, like unresolved conflicts?  The `getPlayerStatement` method for example is definitely there: https://github.com/akasolace/HO/blame/master/src/main/java/core/model/player/Player.java#L2338
